### PR TITLE
Log all HTTP requests with gorilla handlers.CombinedLoggingHandler

### PR DESCRIPTION
**Pull request by jessepeterson**
This should also support #85.

### CODE REVIEW

**Comment by jessepeterson** [comment]
_Supports_ 85, but doesn't solve it — lots more to consistent logging than this. :)

**Comment by groob** [comment]
This is a good start for sure 👍 

I'd like to eventually have all logging be key/value with https://github.com/go-kit/kit/tree/master/log#contextual-loggers

But the gorilla handlers are great for familiar http logs.

File: serve.go [hunk 1]
```diff
@@ -177,6 +178,7 @@ func serve(args []string) error {
 	} else {
 		handler = r
 	}
+	handler = handlers.CombinedLoggingHandler(os.Stdout, handler)
```
Comment (groob) [inline serve.go]: The other logger here goes to stderr
https://github.com/micromdm/micromdm/blob/472d416791f9ef822a6601763b0a6ffbc7c4b022/serve.go#L80

Thoughts on one or the other?

**Comment by jessepeterson** [comment]
Yep. Part of the thought process is to help folks wrap heads around the typical MDM exchanges of enrollment, checkin, etc. Familiar-looking logs helps that. :)

File: serve.go (hunk 1 above)
Comment (jessepeterson) [inline serve.go]: Yeah, I saw that. Generally stdout = normal, stderr = errors. But for a long-running daemon those conventions are sorta.. I dunno.

Eventually of course the logging framework should be flexible enough to log to file, network, API, blah blah blah.

**Pull request merged**